bee-tangle/src/lib.rs

```diff
@@ -0,0 +1,64 @@
+// Copyright 2020-2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+//! A crate that contains foundational building blocks for the IOTA Tangle.
+
+#![deny(missing_docs)]
+
+/// Types used for tangle configuration.
+pub mod config;
+/// Types that represent milestone events.
```

Comment:
```suggestion
/// Types that represent tangle events.
```
Will not necessarily always be related to milestones.